client/errs/errno.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 PingCAP, Inc.
```

Comment:
Should be like this?

```suggestion
// Copyright 2022 TiKV Project Authors.
```